src/idea.js

```diff
@@ -1,16 +1,18 @@
 class Idea {
-  constructor(title, body){
-    this.id = Date.now();
+  constructor(title, body, id, star){
```

Comment:
This is such a clear way to allow us to create new instances while reloading from local storage.